show_notes/Episode_19_Part2.md

```diff
@@ -0,0 +1,26 @@
+# Episode 19, Part 2: Model-View-Intent with Benoît Quenaudon from Square
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+* [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/95)
+
+In the second part of episode #19 Artur and Hannes continue the discussion with Benoît Quenaudon from Square about Model-View-Intent. 
+We discussed the difference between MVP, MVVM and MVI, how to implement MVI, reusable components and how to test MVI based applications.
+
+### Guest:
+- Benoît Quenaudon [@oldergod](https://twitter.com/oldergod), [personal blog](https://benoitquenaudon.com), [GitHub](https://github.com/oldergod)
+
+#### Hosts:
+
+- Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+- Artur Dryomov [@arturdryomov](https://twitter.com/arturdryomov), [personal blog](https://arturdryomov.online/), [GitHub](https://github.com/ming13)
+  
+
+#### Links
+
+- [Cycle.js](https://cycle.js.org)
+- [Android Blueprint: ToDo App using MVI by Benoît](https://github.com/oldergod/android-architecture)
+- [droidcon NYC 2017 - Model-View-Intent for Android](https://www.youtube.com/watch?v=PXBXcHQeDLE)
+- [In-App Navigation with Coordinators](http://hannesdorfmann.com/android/mosby3-mvi-8)
+- [droidcon SF 2017 - The Reactive Workflow Pattern Update](https://www.youtube.com/watch?v=mvBVkU2mCF4)
+- [Benoît's Tweet](https://twitter.com/oldergod/status/999638960384233474): "We see more and more "reactive" architectures on Android but I think you're doing it wrong if you have a `subscribe` in your presenter. Ideally, the presenter should be an ObservableTransformer."
```

Comment:
I think we can add this link — https://staltz.com/unidirectional-user-interface-architectures.html

Reply:
Added 👍